modules/multi-region-cluster/main.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_memorydb_multi_region_cluster" "this" {
```

Comment:
we should leave this sub-module out for now - we don't want to re-wrap just a single resource, it doesn't really provide any value. for now, I think the state of multi-region cluster would be this standalone resource plus multiple instantiations of the memorydb module